src/components/TagActions/TagActions.tsx

```
@@ -384,22 +449,23 @@
+                            <span className="me-1">(</span>
+                            {tag.ancestorTags.map((ancestorTag) => (
                               <span
-                                key={ancestorTag._id}
-                                className="ms-2 my-0"
+                                key={ancestorTag.id}
+                                className="d-flex align-items-center ms-1 my-0"
                                 data-testid="ancestorTagsBreadCrumbs"
                               >
                                 {ancestorTag.name}
-                                <i className="ms-2 fa fa-caret-right" />
+                                {tag.ancestorTags!.indexOf(ancestorTag) !==
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)